src/main/scala/com/typesafe/sbt/packager/rpm/RpmMetadata.scala

```diff
@@ -101,6 +107,22 @@ case class RpmScripts(
 
 }
 
+object RpmScripts {
+
+  import RpmPlugin.Names._
```

Comment:
![Codacy](https://www.codacy.com/versioned/images/favicon.png) Issue found: [Put imports in the beginning of the package](https://www.codacy.com/app/nepomukseiler/sbt-native-packager/file/1559303177/issues/source?bid=2040925&fileBranchId=2489531#l112)